server/server.js

```diff
@@ -177,12 +176,18 @@ function sendCorsHeaders(req : ExpressRequest, res : ExpressResponse) {
     if (corsRequestMethod) {
         res.header(HTTP_HEADER.ACCESS_CONTROL_ALLOW_METHODS, corsRequestMethod);
     }
+
+    res.header(HTTP_HEADER.ACCESS_CONTROL_ALLOW_CREDENTIALS, 'true');
```

Comment:
Should we only set this response header if the `content-type` is `text/plain`?

Reply:
I'm happy to set it all the time, rather than making it a special case 👍 there should be no credentialed data sent from this endpoint, so we're good.